docs/data-sources/dns_acls.md

```diff
@@ -0,0 +1,110 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_dns_acls Data Source - terraform-provider-bloxone"
+subcategory: "DNS"
+description: |-
+  Retrieves information about existing Authoritative DNS ACLs.
+---
+
+# bloxone_dns_acls (Data Source)
+
+Retrieves information about existing Authoritative DNS ACLs.
```

Comment:
@mathewab Can you suggest this in the corresponding service file? It would make it easier for the PR creator to make the change than figuring it out :). Just a thought

Reply:
Ahh, my comment vanished. @mathewab Please add the suggested change in the corresponding service file. Would make it easier for the addressee. Just a thought :)

Reply:
Addressed